sql/database.sql

```diff
@@ -4431,6 +4431,7 @@ CREATE TABLE `patient_data` (
   `migrantseasonal` varchar(255) NOT NULL default '',
   `family_size` varchar(255) NOT NULL default '',
   `monthly_income` varchar(255) NOT NULL default '',
+  `billing_note` varchar(255) NOT NULL default '',
```

Comment:
I wouldn't remove it(it will just no longer function as a place to hold the billing note).